packages/property-grid/src/components/PropertyGrid.scss

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) 2020 Bentley Systems, Incorporated. All rights reserved.
+* Licensed under the MIT License. See LICENSE.md in the project root for license terms.
+*--------------------------------------------------------------------------------------------*/
+
+@import "~@bentley/ui-core/lib/ui-core/style/index";
+@import "~@bentley/ui-core/lib/ui-core/text/index";
+
+$light-background-3: #eef0f3;
+$light-background-5: #c7ccd1;
+$light-foreground-body: #000;
+$light-foreground-primary: #008be1;
```

Comment:
do we need specific values for dark mode?

Reply:
changed these to buic colors